chat-plugins/info.js

```diff
@@ -1874,6 +1874,15 @@ exports.commands = {
 	},
 	pickrandomhelp: ["/pick [option], [option], ... - Randomly selects an item from a list containing 2 or more elements."],
 
+	helixfossil: 'm8b',
+	helix: 'm8b',
+	magic8ball: 'm8b',
+	m8b: function (target, room, user) {
+		if (!this.runBroadcast()) return;
+		let results = ['Signs point to yes.', 'Yes.', 'Reply hazy, try again.', 'Without a doubt.', 'My sources say no.', 'As I see it, yes.', 'You may rely on it.', 'Concentrate and ask again.', 'Outlook not so good.', 'It is decidedly so.', 'Better not tell you now.', 'Very doubtful.', 'Yes - definitely.', 'It is certain.', 'Cannot predict now.', 'Most likely.', 'Ask again later.', 'My reply is no.', 'Outlook good.', 'Don\'t count on it.'];
+		return this.sendReplyBox(results[Math.floor(20 * Math.random())]);
```

Comment:
``20``? I didn't count how long ``results`` is, but I would just do ``results.length`` here; that's more readable.

also, lol this command